packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2380,18 +2380,30 @@ export class StripeHelper extends StripeHelperBase {
       (line) => line.type === 'subscription'
     );
 
-    if (!subscriptionLineItem) {
-      // No subscription is present for the invoice. This should never happen
+    // In certain instances the invoice won't have a 'subscription' line item.
+    // In those cases, select the 'invoiceitem' without proration_details.credited_items
+    const invoiceitemLineItem = !subscriptionLineItem
+      ? invoice.lines.data.find(
+          (line) =>
+            line.type === 'invoiceitem' &&
+            !line.proration_details?.credited_items
```

Comment:
What is this second condition about? I understand proration could be in the form of a credit or a charge. From the [STR](https://mozilla-hub.atlassian.net/browse/FXA-5178?focusedCommentId=605803), the invoice would have two line items, one being a credit proration and the other being a charge proration ([example invoice](https://dashboard.stripe.com/test/invoices/in_1LwxcaBVqmGyQTMaxpoC6CyW)). 🤔

Reply:
This was mostly done to match [a change made a few months ago](https://github.com/mozilla/fxa/pull/14013), to ensure that we get the "latest" plan from the invoice, since there are multiple lines with different plans.

This shouldn't really matter, since the `plan` isn't used in cases where there isn't an `subscription` line item on the invoice, but better safe than sorry.